src/ResponseValidator.ts

```diff
@@ -171,9 +171,6 @@ export class ResponseValidator {
             logger.throw(new Error("Expected code in response"));
         }
 
```

Comment:
here I was not sure what to do here I did not see a reason to save in state the pkcedisabled option,
instead if code verifier does not exist on client I do not check it. I can say for sure that a pkce enabled initiated login will not succeed at server side if you don't send the code, so I think this is safe to remove let me know if you think I'm wring.